test/Interop/Cxx/constructor/Inputs/basic.h

```diff
@@ -0,0 +1,36 @@
+#ifndef TEST_INTEROP_CXX_CONSTRUCTOR_INPUTS_BASIC_H
+#define TEST_INTEROP_CXX_CONSTRUCTOR_INPUTS_BASIC_H
```

Comment:
I don't see any reason not to have them. It prevents errors (that might be kind of hard to figure out) if we ever try to import this into another header for some reason.

Reply:
This header has a module map, so if we include it another header, it will load the corresponding module (not include the header textually). This means that include guards aren't needed, and I haven't seen include guards elsewhere in Swift tests.

Reply:
Fair enough. Done.